cfgov/retirement_api/utils/ss_update_stats.py

```diff
@@ -29,34 +29,34 @@
 outjson = "%s/early_penalty_%s.json" % (data_dir, TODAY.year)
 
 ss_table_urls = {
-    "cola": "http://www.socialsecurity.gov/OACT/COLA/colaseries.html",
-    "actuarial_life": "http://www.socialsecurity.gov/OACT/STATS/table4c6.html",
+    "cola": "https://www.ssa.gov/OACT/COLA/colaseries.html",
+    "actuarial_life": "https://www.ssa.gov/OACT/STATS/table4c6.html",
     # handled by get_retirement_age()
-    "retirement_ages": "http://www.socialsecurity.gov/OACT/ProgData/nra.html",
+    "retirement_ages": "https://www.ssa.gov/OACT/ProgData/nra.html",
     # not needed if we use SS calculator
-    "benefit_bases": "http://www.socialsecurity.gov/OACT/COLA/cbb.html",
-    "delay_credits": "http://www.socialsecurity.gov/retire2/delayret.htm",
-    "awi_series": "http://www.socialsecurity.gov/OACT/COLA/AWI.html",
+    "benefit_bases": "https://www.ssa.gov/OACT/COLA/cbb.html",
+    "delay_credits": "https://www.ssa.gov/retire2/delayret.htm",
+    "awi_series": "https://www.ssa.gov/OACT/COLA/AWI.html",
     # not needed if we use SS calculator
-    "bend_points": "http://www.socialsecurity.gov/OACT/COLA/bendpoints.html",
+    "bend_points": "https://www.ssa.gov/OACT/COLA/bendpoints.html",
     # useful as viz
-    "early_retirement_example": "http://www.socialsecurity.gov/OACT/quickcalc/earlyretire.html",  # noqa: B950
+    "early_retirement_example": "https://www.ssa.gov/OACT/quickcalc/earlyretire.html",  # noqa: B950
     # info only
-    "explainer of AMI calculations": "http://www.socialsecurity.gov/OACT/COLA/piaformula.html",  # noqa: B950
+    "explainer of AMI calculations": "https://www.ssa.gov/OACT/COLA/piaformula.html",  # noqa: B950
     # explanation of terms; info only
-    "benefit_terms": "http://www.socialsecurity.gov/OACT/COLA/Benefits.html#aime",  # noqa: B950
+    "benefit_terms": "https://www.ssa.gov/OACT/COLA/Benefits.html#aime",  # noqa: B950
     # out of scope: rules for achieving 40 work credits (10 years of work);
     # not envisioned for app
-    "credit_rules": "http://www.socialsecurity.gov/planners/retire/credits2.html",  # noqa: B950
+    "credit_rules": "https://www.ssa.gov/planners/retire/credits2.html",  # noqa: B950
     # out of scope: basic work-credit unit to determine whether a worker is
     # covered by SS; you can earn 4 credits a year
-    "quarter_of_coverage": "http://www.socialsecurity.gov/OACT/COLA/QC.html",
+    "quarter_of_coverage": "https://www.ssa.gov/OACT/COLA/QC.html",
     # out of scope:
     # historical and projected male/female death probability tables
-    "death_probabilities": "http://www.socialsecurity.gov/OACT/HistEst/DeathProbabilities2014.html",  # noqa: B950
+    "death_probabilities": "https://www.ssa.gov/OACT/HistEst/DeathProbabilities2014.html",  # noqa: B950
```

Comment:
This one is not found

Reply:
Changed to something that seemed relevant in d1b0b84865c5766e91825c79f5fb3aff10179eb9